scipy/optimize/tests/test_zeros.py

```diff
@@ -294,15 +306,113 @@ def test_newton_full_output(self):
                 # validate the start of the message.
                 with pytest.raises(
                         RuntimeError,
-                        match='Failed to converge after %d iterations, value is .*' % (iters)):
-                    x, r = zeros.newton(self.f1, x0, maxiter=iters, disp=True, **kwargs)
+                        match='Failed to converge after %d iterations, '
+                              'value is .*' % (iters)):
+                    x, r = zeros.newton(
+                        self.f1, x0, maxiter=iters, disp=True, **kwargs)
 
     def test_deriv_zero_warning(self):
         func = lambda x: x**2 - 2.0
         dfunc = lambda x: 2*x
         assert_warns(RuntimeWarning, cc.newton, func, 0.0, dfunc)
 
 
+class TestLowLevelCallable(object):
+    class N_AND_A(ctypes.Structure):
+        _fields_ = [("a", ctypes.c_double), ("n", ctypes.c_int)]
+
+        def __init__(self, n, a):
+            self.n = n
+            self.a = a
+
+        def __repr__(self):
+            return ('N_AND_A(%d, %s)' % (self.n, str(self.a)))
+
+    @classmethod
+    def setup_voidstar(cls):
+        # It should be possible to create a LowLevelCallable and
+        # use cls.n_and_a as a void *
+        if 0:
```

Comment:
Commented-out code should be removed.

Reply:
... or made to work and uncommented.  Working with LowLevelCallables isn't always straight forward.